storage/innobase/srv/srv0srv.cc

```diff
@@ -2754,6 +2767,9 @@ DECLARE_THREAD(srv_master_thread)(
 		} else {
 			srv_master_do_idle_tasks();
 		}
+
+		/* Enable row log encryption if it is set */
+		log_tmp_enable_encryption_if_set();
```

Comment:
The corresponding plugin variable is not dynamic - shouldn't this be one-time initialisation thing?